commands/game/snap.js

```diff
@@ -3,8 +3,8 @@ const { getEmoji } = require("../../config")
 
 module.exports = {
     name: "snap",
-    description: "Kill all players you disguised before.",
-    aliases: ["clap"],
+    description: "Kill all players you have disguised.",
+    aliases: ["clap"],["comeinlikeawreckingball"],
```

Comment:
["clap", "comeinbla"]

Reply:
Bla meaning finish the sentence and remove the comma or shorten the alias?

Reply:
i just didn't want to type the whole thing. the formatting needs to be `[` and `]` on the very ends, and separated by commas inside

Reply:
Gimme a sec

Reply:
I'll fix that